openpype/hosts/houdini/hooks/set_default_display_and_view.py

```diff
@@ -0,0 +1,53 @@
+from openpype.lib.applications import PreLaunchHook, LaunchTypes
+from openpype.settings import get_project_settings
+
+
+class SetDefaultDislayView(PreLaunchHook):
+    """Set default view and default display for houdini host that use OpenColorIO.
```

Comment:
Shouldn't the Hound complain about this line length?

Reply:
yes, it should have! this about 83 characters long!

Reply:
```suggestion
    """Set default view and default display for houdini via OpenColorIO.
```